ivadomed/loader/utils.py

```diff
@@ -107,12 +108,24 @@ def get_new_subject_split(path_folder, center_test, split_method, random_seed,
         log_directory (string): Output folder.
         balance (string): Metadata contained in "participants.tsv" file with categorical values. Each category will be
         evenly distributed in the training, validation and testing datasets.
+        subject_selection (dict): Used to specify a custom subject selection from a dataset.
 
     Returns:
         list, list list: Training, validation and testing subjects lists.
     """
     # read participants.tsv as pandas dataframe
     df = bids.BIDS(path_folder).participants.content
+    if subject_selection is not None:
+        # Verify subject_selection format
+        if not (len(subject_selection["metadata"]) == len(subject_selection["n"]) == len(subject_selection["value"])):
+            raise ValueError("All lists in subject_selection parameter should have the same length.")
```

Comment:
Should we add a test about the presence of the correct keys in the dict? ie metadata, n , value
To be done here or in the config_manager?

Reply:
The keys are added in the config_manager: `"subject_selection": {"n": [], "metadata": [], "value": []}` when keys are missing :3

Reply:
Documentation added here :) 8a8bd78